features/user_can_view_the_full_article.feature

```diff
@@ -18,4 +18,4 @@ Feature: User can view the full article
         Then I am on the "This is so sad" page
         And I should see "This is so sad"
         And I should see "A recent report suggest that news are mostly sad. Which is sad"
-        And I should see "2018-08-17"
\ No newline at end of file
+        And I should see "2018-08-18"
```

Comment:
Haha.... Will we update this every day?

Reply:
We have it as a chore to edit it every day :)

Reply:
This has been addressed in a separate PR.